Assets/Scripts/FPSController.cs

```diff
@@ -42,6 +47,24 @@ void Update()
     void FixedUpdate()
     {
         Jump();
+        Crouch();
+    }
+
+    private void Crouch()
+    {
+        if (Input.GetKey(KeyCode.LeftControl))
```

Comment:
if the if statement only has one line, consider removing the brackets to keep code compact